README.md

```diff
@@ -47,7 +47,7 @@ If you use this paper in academic research, you may cite the following:
 
 ```
 @article{carlini2019evaluating,
-  title={},
+  title={On Evaluating Adversarial Robustness},
```

Comment:
The current README already has this change I believe.

Reply:
Yep indeed. Seems like my page hadn't refreshed properly

Reply:
It's strange that the merge would work, right? Shouldn't this conflict?

Reply:
Trying it out locally, it seems to work fine:

```
$ git merge ftramer/patch-1
Auto-merging README.md
Merge made by the 'recursive' strategy.
 README.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
```

Reply:
Given that it's the exact same change there's no actual conflict, right? Git usually merges these fine (as above in Anish's attempt).